src/js/index.js

```diff
@@ -0,0 +1,43 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const btn = document.getElementById('btn');
+const storageList = document.getElementById('storage_list');
+if (!localStorage.getItem('count')) {
+  localStorage.setItem('count', 0);
+}
+if (!localStorage.getItem('books')) {
+  localStorage.setItem('books', JSON.stringify([]));
+}
+btn.onclick = () => {
+  const books = JSON.parse(localStorage.getItem('books'));
+  let counter = JSON.parse(localStorage.getItem('count'));
+  const book = {
+    id: (counter += 1),
+    title: title.value,
+    author: author.value,
+  };
+  books.push(book);
+  localStorage.setItem('count', counter);
+  localStorage.setItem('books', JSON.stringify(books));
+  window.location.reload();
+};
+/* eslint-disable */ 
+const remove = (id) => {
+  const books = JSON.parse(localStorage.getItem('books'));
+  const newBooks = books.filter((e) => e.id.toString() !== id.toString());
+  localStorage.setItem('books', JSON.stringify(newBooks));
+  window.location.reload();
+};
+/* eslint-enable */
```

Comment:
- [ ] Kindly avoid disabling the entire eslint rule. To improve the professionalism of your project consider fixing the linting issue or simply disable the specific rule.